tests/operator_test.go

```diff
@@ -873,6 +873,18 @@ spec:
 			By("Sanity Checking Deployments infrastructure is deleted")
 			sanityCheckDeploymentsDeleted()
 
+			By("ensuring that namespaces can be successfully created and deleted")
```

Comment:
This should in future prevent that we leave anything behind which can block namespace management.